### DISCUSSION TITLE
add style option

### DISCUSSION THREAD

**Discussion by tfaureperso** [mcanouil/quarto-iconify#59] [Ideas]
for example changing the color of the icon

modif to the iconify.lua file
```
      local style = pandoc.utils.stringify(kwargs["style"])
      if not is_empty(style) then
        attributes = attributes .. ' style="' .. style .. '"'
      end
```
usage example : `{{< iconify icon-park-solid people style="color: #099;">}}`
to change the icon color

**Comment by mcanouil** [comment]
Thanks for the suggestion, but this does not bring much and I would like to keep the extension as close as possible to the Iconify API.

Simply use span divs if you want to style the icons:
```
[{{< iconify fa6-brands apple >}}]{style="color: red;"}
````
